### PR TITLE
log4j2 props, Vert.x 4.0.2 MODAT-98

### PR DESCRIPTION
**Pull request by adamdickmeiss**
Some updates to log4j2 calls as well with {}-formatting.

### CODE REVIEW

File: src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java [hunk 1]
```diff
@@ -185,7 +186,7 @@ private TokenCreator lookupTokenCreator(String passPhrase) throws JOSEException
 
   private void handleSignEncryptedToken(RoutingContext ctx) {
     try {
-      logger.debug("Encrypted token signing request from " + ctx.request().absoluteURI());
+      logger.debug("Encrypted token signing request from {}", () -> ctx.request().absoluteURI());
```
Comment (hjiebsco) [inline src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java:189]: Not consistent. Other logging statements do not use `() ->`

**Review by hjiebsco — APPROVED** [review]

File: src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java (hunk 1 above)
Comment (adamdickmeiss) [reply src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java:189]: Right. Not elegant. Go through the rest or revert this one?

Comment (hjiebsco) [reply src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java:189]: I vote changing this one to be consistent with others. The use of `{}` already makes the string construction delayed in log4j2. See https://logging.apache.org/log4j/2.x/manual/api.html

Comment (adamdickmeiss) [reply src/main/java/org/folio/auth/authtokenmodule/MainVerticle.java:189]: ok.

**Review by hjiebsco — APPROVED** [review]

**Pull request merged**